pkg/controller/machine/metrics.go

```diff
@@ -41,6 +41,23 @@ const (
 // NewMachineControllerMetrics creates new MachineControllerMetrics
 // with default values initialized, so metrics always show up.
 func NewMachineControllerMetrics() *MetricsCollection {
+
+	// numpy.logspace(5,10,10, base=2)
+	// 32sec to ~17min
+	// durations getting observed in milliseconds
+	provisioningBuckets := []float64{
+		32 * 1000,
```

Comment:
Do we need millisecond precision? I think seconds should be enough with this scale.

Reply:
I totally agree, this was due to a misunderstanding on my end. I thought all timestamps returned by `/metrics` were milliseconds, but actually they are in seconds, my bad.
Moved to seconds in the most recent commit